src/renderers/shared/reconciler/ReactMultiChild.js

```diff
@@ -218,31 +219,43 @@ var ReactMultiChild = {
      * of `ReactDOMComponent`, a mount image is a string of markup.
      *
      * @param {?object} nestedChildren Nested child maps.
+     * @param {ReactReconcileTransaction|ReactServerRenderingTransaction} transaction
+     * @param {object} context
+     * @param {?DOMNode} when reconnecting to server markup, the DOM node to reuse.
```

Comment:
add the variableName.  Also, this function takes 5 arguments, but only 4 of them are documented.